samples/client/petstore/typescript-node/api.ts

```diff
@@ -25,6 +25,20 @@ export class Dog extends Animal {
     "breed": string;
 }
 
+export class FormatTest {
```

Comment:
It's for testing different type and format. 

Please update typescript script to use `modules/swagger-codegen/src/test/resources/2_0/petstore.yaml` (the original Petstore spec) for the time being.